WordPress/gutenberg
# NavigableContainer: use code instead of keyCode for keyboard events

## What?


- Refactor the `NavigableContainer` component to rely on `code` instead of `keyCode` for keyboard events
- Refactor unit tests to modern RTL and `user-event`
- Fix a bug in `TabbableContainer` where pressing `tab` or `shift tab` would end up skipping one tabbable element (because both the component and the browser would move focus to the next tabbable)
- Added Storybook examples


Note that `TabbableContainer` doesn't seem to be used in the repo.

## Why?


[`keyCode`](https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/keyCode) is deprecated, and replaced by [`code`](https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/code)

## Testing Instructions





Interact with the Storybook examples, make sure that the components behave as expected.
Inspect the unit tests, make sure that:
 - they replace fully the previous unit tests
 - their code makes sense
